tests/device.test.js

```diff
@@ -229,11 +242,19 @@ describe("Send to-device message", () => {
 });
 
 describe("Key Verification", () => {
-    const userId1 = new UserId("@alice:example.org");
-    const deviceId1 = new DeviceId("alice_device");
+    let userId1;
+    let deviceId1;
```

Comment:
Those kind of changes are because we don't load the WASM module synchronously anymore, so accessing them before it's loaded (so before the `beforeAll` call in the `helper.js`)

Reply:
👍

Reply:
Are these changes still necessary, now that the package works if you try to use it synchronously (at least in nodejs) ?